packages/app/src/client/services/Picker/types.ts

```diff
@@ -18,6 +18,7 @@ import { ReactNode } from "react";
 export type PickerOption = {
   id: string;
   text: string;
+  data?: any;
```

Comment:
@terrcin  Not important to block this PR, but in general we should try to avoid the `any` type because anything that relies on it downstream will not be type-checked by the compiler for correctness. We lose a lot of static analysis value at build time. Perhaps a generic could be used here